src/public/monetization.ts

```diff
@@ -0,0 +1,37 @@
+import { sendMessageToParent } from '../internal/communication';
```

Comment:
is this file already merged to master ?

Reply:
Yes. ../internal/communication is an existing file.